include/aspect/postprocess/visualization/devstress.h

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
I think you included the files twice, once as `devstress.h` and once as `deviatoric_stress.h`, same for the .cc files.

Reply:
you are right: shear stress does exactly the same. So no need anymore for this postprocessor